nexus/external-api/src/lib.rs

```diff
@@ -1117,6 +1117,19 @@ pub trait NexusExternalApi {
         path_params: Path<params::InstancePath>,
     ) -> Result<HttpResponseDeleted, HttpError>;
 
+    /// Update instance
+    #[endpoint {
+        method = PUT,
+        path = "/v1/instances/{instance}",
+        tags = ["instances"],
+    }]
+    async fn instance_update(
```

Comment:
(keeping this one `update` since its parameters are `*Update` and i expect that's what searchers would be looking for when skimming the file. if someone gets lost because of this in the future, i'm very sorry!!)